### PR TITLE
PAYARA-3510 Add support for JDKs > 1.6 to cmp-enhancer

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Pandrex247** [comment]
Jenkins test please

File: appserver/persistence/cmp/enhancer/src/main/java/com/sun/jdo/api/persistence/enhancer/classfile/ClassFile.java [hunk 1]
```diff
@@ -65,10 +65,16 @@
...
     new short[]{48, 0}, // jdk 1.4
     new short[]{49, 0}, // jdk 1.5
-    new short[]{50, 0}  // jdk 1.6
+    new short[]{50, 0}, // jdk 1.6
+    new short[]{51, 0}, // jdk 1.7
+    new short[]{52, 0}, // jdk 1.8
+    new short[]{53, 0}, // jdk 9
+    new short[]{54, 0}, // jdk 10
+    new short[]{55, 0}, // jdk 11
+    new short[]{50, 0}, // jdk 12
```
Comment (svendiedrichsen) [inline appserver/persistence/cmp/enhancer/src/main/java/com/sun/jdo/api/persistence/enhancer/classfile/ClassFile.java]: This looks like JDK 1.6.

File: appserver/persistence/cmp/enhancer/src/main/java/com/sun/jdo/api/persistence/enhancer/classfile/ClassFile.java [hunk 2]
```diff
@@ -65,10 +65,16 @@
...
+    new short[]{56, 0}  // jdk 13
```
Comment (svendiedrichsen) [inline appserver/persistence/cmp/enhancer/src/main/java/com/sun/jdo/api/persistence/enhancer/classfile/ClassFile.java]: This should be 57,0

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**